client/src/containers/datasets/layers/projects/legend.tsx

```diff
@@ -1,3 +1,12 @@
-export default function Legend() {
-  return <div className="h-3 w-3 rounded-full bg-green-500" />;
+import LegendSettings from '@/containers/legend-settings';
+import { LayerSettings } from '@/types/layers';
+
+export default function Legend({ settings }: { settings: LayerSettings }) {
+  return (
+    <div className="flex items-center space-x-2">
+      <div className="h-3 w-3 rounded-full bg-green-500" />
+      <p className="text-xs capitalize">{settings.id}</p>
+      <LegendSettings settings={settings} />
```

Comment:
Think we need a justify between here
![Screenshot 2024-03-21 at 11 22 32](https://github.com/Vizzuality/afoco/assets/51995866/242bc330-7cf7-42b0-8258-292b7d78ef50)